pramen-py/src/pramen_py/metastore/reader.py

```diff
@@ -184,6 +283,8 @@ def get_table(
         """
 
         metastore_table = get_metastore_table(table_name, self.tables)
+        self.validate_partitions_paths(metastore_table)
```

Comment:
We shouldn't do it for all formats. And for get_table() we should not do this at all.

Reply:
```
paths = get_partition_paths(metastore_table)
validate_partition_paths(paths, metastore_table)
dates_list = _extract_dates_from_paths(paths, metastore_table)
if dates_list is empty:
  // error
filtered_dates_list = filter_dates(...)
if filtered_dates_list is empty:
  // error
return filtered_dates_list
```